starter_scripts/exaitde_within_docker_container_slim.sh

```diff
@@ -70,11 +71,12 @@ function create_env_file_debug_protected() {
 }
 
 old_umask=$(umask)
-umask 277
+umask 077
```

Comment:
Same deal as above, if you need the create a file with a specific `umask` you may also extract it into a function. Each time there is new code changing the `umask` manually back and forth it increases the change that reverting it back is forgotten.
(What happens if somebody else adds code in between etc. ...)

The function interface could take a list of files and a umask.

Reply:
As mentioned above, reverting the `umask` is actually a "nice to have".